components/footer/footer-group.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import useTheme from '../use-theme'
+import { useFooterContext } from './footer-context'
+
+interface Props {
+  title: string | React.ReactNode
+}
+
+const defaultProps = {
+  title: '',
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type FooterGroupProps = Props & typeof defaultProps & NativeAttrs
+
+const FooterGroup: React.FC<React.PropsWithChildren<FooterGroupProps>> = ({
+  title,
+  children,
+  ...props
+}) => {
+  const theme = useTheme()
+  const { breakPoint } = useFooterContext()
```

Comment:
We have hooks on `breakpoints`.
But we can't use the hooks directly internally, this may lead to inconsistent data when rendering on the server-side, please refer here: #258 

Referring to the design of the `Grid` component, we need to integrate the breakpoints in the style.